blocks/trending-news/trending-news.js

```diff
@@ -51,9 +64,9 @@ function createNewsCards(news) {
   const iconSpan = document.createElement('span');
   iconSpan.className = 'icon icon-icon-time';
   const abbr = document.createElement('abbr');
-  abbr.textContent = `${news.date} `;
+  abbr.textContent = `${item.PublishedOn} `;
   const abbrSource = document.createElement('abbr');
-  abbrSource.textContent = news.source;
+  abbrSource.textContent = item.Author;
```

Comment:
@ravverma do we get this information from API ?

Reply:
yes, Author is in the API response.